src/handlers/opensea-api.ts

```diff
@@ -6,38 +6,53 @@ import {
   NFT_API_URL,
 } from 'react-native-dotenv';
 import { rainbowFetch } from '../rainbow-fetch';
-import NetworkTypes from '@rainbow-me/networkTypes';
-import { parseAccountUniqueTokens } from '@rainbow-me/parsers';
+import NetworkTypes, { Network } from '@rainbow-me/networkTypes';
```

Comment:
not a blocker but for follow up:
`NetworkTypes` can now be fully replaced with `Network` as this was actually just used for JS files